src/components/application_manager/include/application_manager/app_launch/app_launch_data_json.h

```diff
@@ -53,7 +52,11 @@ class AppLaunchDataJson : public AppLaunchDataImpl {
    * @brief Constructor of AppLaunchDataJson object
    */
   AppLaunchDataJson(const AppLaunchSettings& settings,
-                    resumption::LastState& last_state);
+                    resumption::LastStateWrapperPtr last_state_wrapper);
+
+  DEPRECATED AppLaunchDataJson(const AppLaunchSettings& settings,
```

Comment:
We put it in new release. Remove deprecated staff

Reply:
@LuxoftAKutsan removed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/4be3a56fad6056c2697510133c5a65a598a2ac85